packages/ui/src/Card.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react'
+import { Icon, IconProps } from './Icon'
+import cn from 'classnames'
+
+import { IconButton, IconButtonProps } from './IconButton'
+
+import styleConsts from '../styles/constants/export.scss'
+import styles from './Card.module.scss'
+
+type CardHeaderProps =
+  | {
+      title: string
+      icon?: IconProps['icon']
+      iconButton?: never
+    }
+  | {
+      title: string
+      iconButton?: Omit<IconButtonProps, 'size' | 'kind'>
+      icon?: never
+    }
+  | {
+      title?: never
+      iconButton?: never
+      icon?: never
+    }
+
+export type CardProps = {
+  type?: 'primary' | 'secondary' | 'highlighter'
+  icon?: IconProps['icon']
+} & CardHeaderProps
```

Comment:
Should we use `DataTestProp` and allow to pass a custom data-test like we decided to do for TextField & Check/Radio buttons? So that the code base is consistent?

Reply:
Sure, didn't know that was a thing, added 👍